include/mailio/message.hpp

```diff
@@ -512,6 +512,21 @@ class MAILIO_EXPORT message : public mime
     **/
     std::multimap<std::string, std::string> headers() const;
 
+    /**
+    @return Message all of the data with no parsing actions
+    **/
+    std::vector<std::string> all_data() const;
```

Comment:
Why do you need it in the raw format? The purpose of the `message` class is to parse the string obtained over network.